templates/products/rdm.html

```diff
@@ -123,6 +123,11 @@ <h3>Community</h3>
         <a href="https://www.wacren.net"><img src="{{ '/static/img/logo-wacren.png'|asseturl }}" title="West and Central African Research and Education Network"></a>
       </div>
     </div>
+    <div class="row centered">
```

Comment:
I think the rest are set on alphabetical order, could you move it to the corresponding place? Let's assume it starts by *U*